test/Microsoft.Data.Sqlite.Tests/SqliteParameterTest.cs

```diff
@@ -390,6 +391,34 @@ public void Bind_with_prefixed_names()
                 }
             }
         }
+        
+        [Fact]
+        public void Add_range_of_parameters_using_DbCommand_base_class()
+        {
+            using (var connection = new SqliteConnection("Data Source=:memory:"))
+            {
+                var command = connection.CreateCommand() as DbCommand;
+                command.CommandText = "SELECT @Value1, @Value2;";
+
+                var parameterValue1 = new SqliteParameter("@Value1", SqliteType.Text);
+                parameterValue1.Value = "ABCDE";
+
+                var parameterValue2 = new SqliteParameter("@Value2", SqliteType.Text);
+                parameterValue2.Value = "FGHIJ";
+
+                var parameters = new[] { parameterValue1, parameterValue2 };
+
+                command.Parameters.AddRange(parameters);
```

Comment:
`parameters` is an `IEnumerable<SqliteParameter>`, so this is not calling the correct overload.

Reply:
Oh wait, command is a `DbCommand`. All is well.